migrations/versions/2ee66194b9eb_remove_run_table.py

```diff
@@ -17,24 +17,36 @@
 
 
 def upgrade():
-    # TODO: data migration from run to benchmark_result goes here!
-
     op.add_column(
         "benchmark_result",
-        sa.Column("run_tags", postgresql.JSONB(astext_type=sa.Text()), nullable=False),
+        sa.Column("run_tags", postgresql.JSONB(astext_type=sa.Text()), nullable=True),
     )
     op.add_column("benchmark_result", sa.Column("run_reason", sa.Text(), nullable=True))
     op.add_column(
         "benchmark_result", sa.Column("commit_id", sa.String(length=50), nullable=True)
     )
     op.add_column(
         "benchmark_result",
-        sa.Column("hardware_id", sa.String(length=50), nullable=False),
+        sa.Column("hardware_id", sa.String(length=50), nullable=True),
     )
     op.drop_constraint("summary_run_id_fkey", "benchmark_result", type_="foreignkey")
     op.create_foreign_key(None, "benchmark_result", "hardware", ["hardware_id"], ["id"])
     op.create_foreign_key(None, "benchmark_result", "commit", ["commit_id"], ["id"])
 
+    op.execute(
+        """
+        UPDATE benchmark_result SET
+            run_tags = jsonb_build_object('name', coalesce(run.name, '')),
```

Comment:
This is fine here, but going forward if `run_name` isn't specified, `name` just isn't in `run_tags`, right? Or `run_tags` is `{"name": null}`? I'd rather store nulls and have the `coalesce()` on the query side if we can

Reply:
Yes, if `run_name` isn't posted with the benchmark result then there's no `name` key in run_tags. I think this is completely fine because I want to phase out run name in favor of users thinking about how to store better run tags.